Makefile

```diff
@@ -81,3 +79,14 @@ test-database-down:
 	docker rm -v boundary-sql-tests || true
 
 .PHONY: testacc tools docs test-database-up test-database-down
+
+.PHONY: copywrite
+copywrite:
+	copywrite headers
+
+.PHONY: fmt
+fmt:
+	gofumpt -w $$(find . -name '*.go')
+
+.PHONY: gen
+gen: docs copywrite
```

Comment:
Do we want to add `fmt` here?

Reply:
ty!

Reply:
Looks like it's failing because it can't find `gofumpt`

Reply:
looking